examples/integrations/langchain/langchain.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Should we link the written guide here as well  <a href="https://labelbox.com/guides/turn-langchain-logs-into-conversational-data-with-labelbox/" target="_blank">associated written guide</a> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>